sample-apps/spark/build.gradle

```diff
@@ -38,9 +38,9 @@ dependencies {
     implementation "io.grpc:grpc-api:1.34.1"
     implementation "io.grpc:grpc-netty-shaded:1.34.1"
 
-    implementation platform("io.opentelemetry:opentelemetry-bom:1.0.1")
+    implementation platform("io.opentelemetry:opentelemetry-bom:1.4.1")
     implementation "io.opentelemetry:opentelemetry-api"
-    implementation "io.opentelemetry:opentelemetry-sdk-metrics:1.0.0-alpha"
+    implementation "io.opentelemetry:opentelemetry-sdk-metrics:1.4.1-alpha"
```

Comment:
I know there are some metric related breaking changes after OTel 1.3.x SDK. Could you help to run `otlp-metric ` test case before merging the change?

Reply:
Or can we split this PR. let's put the SDK upgrade into a separate PR?

Reply:
Sounds good. I'll separate and test it.